### DISCUSSION TITLE
F-Droid OsmAnd~ release plan

### DISCUSSION THREAD

**Discussion by fschn90** [osmandapp/OsmAnd#16888] [General]
Hi, thanks for the great work. I really enjoy the app and use it a lot.

I installed [OsmAnd~ from the F-Droid](https://f-droid.org/packages/net.osmand.plus/) app store. Currently the latest version is 4.3.12 (431203). 

Is there a release plan for when the more up to date versions are to be included on F-Droid? In particular OsmAnd~ 4.4.0#14201mqta, Released: 2023-02-25?

Much appreciated & best, Flo

**Comment by mikehgentry** [comment]
The F-Droid version is compiled from source by independent people, so you'd have to ask them

**Comment by fschn90** [comment]
In case someone else is interested, following ticket has some comments regarding version 4.4:
https://gitlab.com/fdroid/fdroiddata/-/issues/2848
Which seems to be an indication that people are working it already.
